alerts/openshift-container-storage-operator/StorageAutoScalingCapacityReached.md

```diff
@@ -0,0 +1,30 @@
+# StorageAutoScalingCapacityReached
+
+## Meaning
+
+The Storage Capacity Limit is the maximum storage that can be provisioned for
+the system. `StorageAutoScalingCapacityReached` alert is fired when the complete
+storage space has been utilized and hence additional storage cannot be allocated
+for the device class in the particular storage cluster.
+
+## Impact
+
+The storage cannot be auto-scaled at the moment.
+
+## Diagnosis
+
+Retrieve status of `storageCapacityLimitReached` from the StorageAutoScaler CR.
+
+```bash
+oc get StorageAutoScaler <resource_name> -n openshift-storage -o json | jq -r '.status.storageCapacityLimitReached'
+```
+
+If status of `storageCapacityLimitReached` is `true` then maximum capacity of
+the storage has been utilized.
+
+## Mitigation
+
+Scale storage: Increase the capacity by specifying it in the command below:
```

Comment:
>Increase the capacity by specifying it in the command below:

Run the below command, specify the new increased StorageCapacityLimit

Reply:
Note: 
`status.error.message` will be specifying the min required increased StorageCapacityLimit, so either at least specify that or more